gulpfile.js

```diff
@@ -0,0 +1,42 @@
+const gulp = require('gulp');
+const fsbx = require("fuse-box");
+const clean = require('gulp-clean');
+const tsc = require('gulp-typescript');
+const sourcemaps = require('gulp-sourcemaps');
+const express = require('express');
+const path = require('path');
+const { dev, storybook } = require('./fuse');
+
+let node;
+
+// TODO: read app name from command line option
+const appName = 'loyalty';
+
+gulp.task('clean', function() {
+    return gulp.src('dist', { read: false })
+        .pipe(clean());
+});
+
+gulp.task('compile-server', () => {
+    const tsProject = tsc.createProject('tsconfig.json', {
+        target: "es5"
+    })
+
+    const tsResult = gulp.src(`src/**/*.{ts,tsx}`)
+        .pipe(sourcemaps.init())
+        .pipe(tsProject())
+
+    return tsResult.js
+        .pipe(sourcemaps.write('.'))
+        .pipe(gulp.dest('dist'))
+})
```

Comment:
This won’t work with isomorphic Relay, as the Babel transform is needed for the queries to work. i.e. the pipeline is:

TypeScript -> JS with unaltered Relay queries -> Babel + Relay plugin -> JS with compiled Relay queries.